dashboard-notifications/src/main/java/uk.gov.hmcts.reform/dashboard/entities/NotificationEntity.java

```diff
@@ -10,33 +12,63 @@
 import javax.persistence.ManyToOne;
 import javax.persistence.Table;
 import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
 
 @lombok.Data
 @lombok.Builder(toBuilder = true)
 @lombok.NoArgsConstructor
-@lombok.AllArgsConstructor
 @Entity
+@AllArgsConstructor
 @Table(name = "dashboard_notifications", schema = "dbs")
 public class NotificationEntity implements Serializable {
 
     private static final long serialVersionUID = -649190928299762655L;
 
     @Id
-    @NotNull
+    @Column(name = "id", nullable = false)
     private UUID id;
-    @ManyToOne
-    @JoinColumn(name = "notifications_Templates_id", referencedColumnName = "id")
-    private NotificationTemplateEntity notificationTemplateEntity;
+
+    @ManyToOne(fetch = javax.persistence.FetchType.LAZY)
+    @JoinColumn(name = "dashboard_notifications_templates_id")
+    private NotificationTemplateEntity dashboardNotificationsTemplates;
+
+    @Size(max = 256)
+    @Column(name = "reference", length = 256)
     private String reference;
-    @Column(name = "notification_name")
-    private String name;
+
+    @Size(max = 256)
+    @Column(name = "notification_name", length = 256)
+    private String notificationName;
+
+    @Size(max = 256)
+    @Column(name = "citizen_role", length = 256)
     private String citizenRole;
-    private String enHTML;
-    private String cyHTML;
-    @Column(name = "message_param")
-    private String params;
+
+    @Size(max = 256)
+    @Column(name = "en_html", length = 256)
+    private String enHtml;
+
+    @Size(max = 256)
+    @Column(name = "cy_html", length = 256)
+    private String cyHtml;
+
+    @Size(max = 256)
+    @Column(name = "message_param", length = 256)
```

Comment:
with that changes we need to update the migration files about create tables....
...created_by for example and every one.

Reply:
Do we? The changes are to the entities, but the database column names should still match due to the @Column.
With the old file, there seemed to be entity/database mismatches.